src/test/Components/CreateImageWizardV2/CreateImageWizard.test.tsx

```diff
@@ -396,1203 +398,1216 @@ describe('Step Upload to AWS', () => {
+  //       name: /select resource group/i,
+  //     });
+  //     await user.click(resourceGroupDropdown);
+  //     await user.click(screen.getByLabelText('Resource group myResourceGroup1'));
+  //     await clickNext();
+
+  //     // Registration
+  //     await screen.findByText(
+  //       'Automatically register and enable advanced capabilities'
+  //     );
+  //     const registerRadio = screen.getByTestId('registration-radio-now');
+  //     expect(registerRadio).toHaveFocus();
+  //     await screen.findByRole('textbox', {
+  //       name: 'Select activation key',
+  //     });
+  //     // skip registration
+  //     const registerLaterRadio = screen.getByTestId('registration-radio-later');
+  //     await user.click(registerLaterRadio);
+
+  //     await clickNext();
+
+  //     // File system configuration
+  //     await clickNext();
+
+  //     // Packages
+  //     const view = screen.getByTestId('search-available-pkgs-input');
+
+  //     const availablePackagesInput = within(view).getByRole('textbox', {
+  //       name: /search input/i,
+  //     });
+  //     await waitFor(() => expect(availablePackagesInput).toBeEnabled());
+  //     expect(availablePackagesInput).toHaveFocus();
+  //     await clickNext();
+
+  //     // TODO: what should have focus on Custom Repos step?
+  //     await clickNext();
+
+  //     // Name
+  //     const nameInput = screen.getByRole('textbox', { name: /image name/i });
+  //     expect(nameInput).toHaveFocus();
+  //     await clickNext();
+  //   });
+
+  //   test('pressing Esc closes the wizard', async () => {
+  //     await setUp();
+  //     // wizard needs to be interacted with for the esc key to work
+  //     await waitFor(
+  //       async () => await user.click(await screen.findByTestId('upload-aws'))
+  //     );
+  //     await user.keyboard('{escape}');
+  //     expect(router.state.location.pathname).toBe('/insights/image-builder');
+  //   });
+
+  //   test('pressing Enter does not advance the wizard', async () => {
+  //     await setUp();
+  //     await waitFor(
+  //       async () => await user.click(await screen.findByTestId('upload-aws'))
+  //     );
+  //     await user.keyboard('{enter}');
+  //     screen.getByRole('heading', {
+  //       name: /image output/i,
+  //     });
+  //   });
+
+  //   test('target environment tiles are keyboard selectable', async () => {
+  //     const testTile = async (tile) => {
+  //       tile.focus();
+  //       await user.keyboard('{space}');
+  //       expect(tile).toHaveClass('pf-m-selected');
+  //       await user.keyboard('{space}');
+  //       expect(tile).not.toHaveClass('pf-m-selected');
+  //     };
+
+  //     await setUp();
+  //     await clickNext();
+
+  //     await waitFor(() => screen.findByTestId('upload-aws'));
+  //     testTile(screen.getByTestId('upload-aws'));
+  //     testTile(screen.getByTestId('upload-google'));
+  //     testTile(screen.getByTestId('upload-azure'));
+  //   });
```

Comment:
these commented lines have been added, can you remove them please?
@regexowl can you confirm that we don't want those?

Reply:
sorry wrong pr...